source/extensions/common/wasm/wasm.cc

```diff
@@ -113,8 +113,7 @@ Wasm::Wasm(absl::string_view runtime, absl::string_view vm_id, absl::string_view
       time_source_(dispatcher.timeSource()), vm_configuration_(vm_configuration),
       wasm_stats_(WasmStats{
           ALL_WASM_STATS(POOL_COUNTER_PREFIX(*scope_, absl::StrCat("wasm.", runtime, ".")),
-                         POOL_GAUGE_PREFIX(*scope_, absl::StrCat("wasm.", runtime, ".")))}),
-      stat_name_set_(scope_->symbolTable().makeSet("Wasm").release()) {
+                         POOL_GAUGE_PREFIX(*scope_, absl::StrCat("wasm.", runtime, ".")))}) {
```

Comment:
I assume it's OK to take a bunch of locks here, as this does not happen in the request path; is that right?

You can easily code around this if you need to, and I've been mulling over creating a new family of stats-macros for instantiating stats without taking locks.

Reply:
Typically this is done in the configure path which is coming out of the dispatcher, so same thread, but not (typically) in the request path.